### PR TITLE
Creates a standard backup job

### PR DESCRIPTION
**Pull request by Vermyndax**

### CODE REVIEW

**Comment by afeld** [comment]
Thoughts about moving this up to the Jenkins role? /cc https://github.com/GSA/jenkins-deploy/issues/9

File: ansible/playbooks/jenkins-master.yml [hunk 1]
```diff
@@ -13,6 +13,17 @@
     - gsa.jenkins
   tasks:
 
+    - name: Set YUM repos for AWSCLI
+      yum_repository:
+        name: rhui-REGION-rhel-server-optional
+        description: rhel optional rpms
+        enabled: yes
+        mirrorlist: https://rhui2-cds01.REGION.aws.ce.redhat.com/pulp/mirror/content/dist/rhel/rhui/server/7/$releasever/$basearch/optional/os
```
Comment (afeld) [inline ansible/playbooks/jenkins-master.yml:21]: Might want to include a comment about where this URL comes from.

File: jenkins-jobs/jenkins-backup/job-tpl.xml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+echo &quot;Uploading archive to S3&quot;
+aws s3 cp jenkins-configuration.tar.gz s3://${jenkins_backup_s3_bucket}/${jenkins_backup_s3_key}/
+
+# Remove tar.gz after it gets uploaded to S3
+rm -rf jenkins-configuration.tar.gz</command>
+    </hudson.tasks.Shell>
+  </builders>
+  <publishers/>
+  <buildWrappers/>
+</project>
```
Comment (afeld) [inline jenkins-jobs/jenkins-backup/job-tpl.xml:71]: Out of curiosity, did you look into any of the plugins/options in https://github.com/GSA/jenkins-deploy/issues/9? Not that it's _super_ complex, but would love to not have to maintain this ourselves.

**Review by afeld — APPROVED** [review]

File: jenkins-jobs/jenkins-backup/job-tpl.xml (hunk 2 above)
Comment (Vermyndax) [reply jenkins-jobs/jenkins-backup/job-tpl.xml:71]: Yep, sure did. They didn’t really match my desired use case to store the backup in S3 without extra configuration anyway, so I didn’t see the point in maintaining a few extra components when a simple bash script could do it all ;)

**Pull request merged**